service/matching/db_task_manager.go

```diff
@@ -187,9 +206,7 @@ func (d *dbTaskManager) isStopped() bool {
 	return atomic.LoadInt32(&d.status) == common.DaemonStatusStopped
 }
 
-func (d *dbTaskManager) acquireLoop(
-	ctx context.Context,
-) {
+func (d *dbTaskManager) acquireLoop(ctx context.Context) error {
```

Comment:
if this function is returning error, then where is the error handled?

Reply:
this function is supposed to be long running, returning an error is not really useful

Reply:
Having error as return value is required to use goro